Source/Upstream: Yes, in v238-stable branch

From 62618a6ef199e2691804c69f76ca52ff470ac538 Mon Sep 17 00:00:00 2001
From: Elia Geretto <elia.f.geretto@gmail.com>
Date: Mon, 19 Mar 2018 08:00:28 +0100
Subject: [PATCH 26/51] hwdb: Split touchpad rules for X550CC and S550C

(cherry picked from commit 80188d5bf6e3a97b5faff3a868ecd2627d94e352)
---
 hwdb/60-evdev.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb/60-evdev.hwdb b/hwdb/60-evdev.hwdb
index 7e4ae61ea..69bd09c7b 100644
--- a/hwdb/60-evdev.hwdb
+++ b/hwdb/60-evdev.hwdb
@@ -115,7 +115,8 @@ evdev:name:ETPS/2 Elantech Touchpad:dmi:bvn*:bvr*:bd*:svnASUSTeKComputerInc.:pnK
  EVDEV_ABS_36=::16
 
 # Asus X550CC and S550CB
-evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pn?550C?:*
+evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnX550CC:*
+evdev:name:ETPS/2 Elantech Touchpad:dmi:*:svnASUSTeKCOMPUTERINC.:pnS550CB:*
  EVDEV_ABS_00=::31
  EVDEV_ABS_01=::30
  EVDEV_ABS_35=::31
-- 
2.16.3

